ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -130,6 +130,61 @@ roles/eos_designs/docs/tables/fabric-topology.md
 roles/eos_designs/docs/tables/fabric-ip-addressing.md
 --8<--
 
+## PREVIEW - Fabric Numbering
```

Comment:
I agree to include "PREVIEW" in the title, but we are inconsistent, i.e. "Zscaler Internet Exit". I think we should update Zscaler title, which can be done in another PR.